src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/lines/AbstractLine.java

```diff
@@ -0,0 +1,28 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays.lines;
+
+import me.filoghost.holographicdisplays.api.hologram.Hologram;
+import org.betonquest.betonquest.compatibility.holographicdisplays.HologramLoop;
+
+/**
+ * Superclass for all hologram content types.
+ * <p>
+ * HologramLoop parses and validates the hologram instructions into instances of AbstractLine
+ * so the content can be updated without dealing with parsing errors again.
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.